libcrux-ml-kem/src/mlkem512.rs

```diff
@@ -43,9 +48,17 @@ pub type MlKem512PublicKey = MlKemPublicKey<CPA_PKE_PUBLIC_KEY_SIZE_512>;
 /// An ML-KEM 512 Key pair
 pub type MlKem512KeyPair = MlKemKeyPair<SECRET_KEY_SIZE_512, CPA_PKE_PUBLIC_KEY_SIZE_512>;
 
+/// An Unpacked ML-KEM 512 Public key
+#[allow(type_alias_bounds)]
+pub type MlKem512PublicKeyUnpacked<Vector: VectorType> = MlKemPublicKeyUnpacked<RANK_512, Vector>;
+/// Am Unpacked ML-KEM 512 Key pair
+#[allow(type_alias_bounds)]
+pub type MlKem512KeyPairUnpacked<Vector: VectorType> = MlKemKeyPairUnpacked<RANK_512, Vector>;
+
 // Instantiate the different functions.
 macro_rules! instantiate {
-    ($modp:ident, $p:path, $doc:expr) => {
+    ($modp:ident, $p:path, $vec:path, $doc:expr) => {
+        /// Provides $modp implementations of ML-KEM 512
```

Comment:
Not sure if this is needed. Putting macro internals in public docs isn't great.
Not that there is a rustdoc in line below, set in the macro.
(same for the other key sizes)

Reply:
Removed